### PR TITLE
New lint and release wraps

### PR DESCRIPTION
**Pull request by TPRobots**
Pull request opened by github-pullrequestcreator.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -268,4 +280,11 @@ You can use pytest to run tests against your current Python version. To run test
...
 
 ## Contributors
-sloev, svdgraaf, simongarnier, elviejokike, paddie, AlbertoTrinidade, martinbuberl
+
+@nabrosimoff, @elviejokike, @eduardomourar, @devgrok, @AlbertoTrindade, @paddie, @svdgraaf, @simongarnier, @martinbuberl, @sloev
+
+**Wanna contribute?**
+
+* crete issues or PR's, no real guidelines given here.
+* @sloev will review and take care of version bumping and release etc
```
Comment (sallas) [inline README.md]: I wouldn't mention myself here, just write a contributed will look at it.

File: README.md [hunk 2]
```diff
@@ -268,4 +280,11 @@ You can use pytest to run tests against your current Python version. To run test
 See [`setup.py`](setup.py) for test dependencies and install them with .[test].
 
 ## Contributors
-sloev, svdgraaf, simongarnier, elviejokike, paddie, AlbertoTrinidade, martinbuberl
+
+@nabrosimoff, @elviejokike, @eduardomourar, @devgrok, @AlbertoTrindade, @paddie, @svdgraaf, @simongarnier, @martinbuberl, @sloev
+
+**Wanna contribute?**
+
+* crete issues or PR's, no real guidelines given here.
```
Comment (sallas) [inline README.md]:
```suggestion
* Create issues or pull requests, no real guidelines given here.
```

File: README.md (hunk 1 above)
Comment (sloev) [reply README.md]: yeah, good idea!

File: setup.py [hunk 3]
```diff
@@ -4,69 +4,46 @@
...
-    long_description=readme + '\n\n' + history,
+    long_description=readme + "\n\n" + history,
+    long_description_content_type="text/markdown",
     author="jgv",
-    author_email='jgv@trustpilot.com',
-    url='https://github.com/trustpilot/python-lambdarest',
-    packages=[
-        'lambdarest',
-    ],
+    author_email="jgv@trustpilot.com",
```
Comment (sallas) [inline setup.py]: You changed this to `__email__ = "johannes.valbjorn@gmail.com"`in __version__.py. Pick one?

Comment (sloev) [reply setup.py]: its because of integration on pypi, it does not redirect to my user otherwise

Comment (sallas) [reply setup.py]: ok

**Pull request merged**